docs/en/docs/deployment/manually.md

```diff
@@ -85,7 +85,7 @@ You can then run your application the same way you have done in the tutorials, b
 
     </div>
 
-!!! warning
+!!! Attention
```

Comment:
```suggestion
!!! warning
```

The specific keyword is what triggers and shows the correct tooltip. Also, this one is for English.

Reply:
Thanks for catching this one!